### PR TITLE
Speed up Builds

### PR DESCRIPTION
**Pull request by andrewstech**
# Description

Me putting a v8 engine into react

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by andrewstech** [comment]
New Build time 15 seconds

Website loading time slightly quicker

File: .github/workflows/package-lock.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
         run: rm package-lock.json
 
       - name: build new one
-        run: npm install
+        run: npm install --legacy-peer-deps
```
Comment (Lukas-Batema) [inline .github/workflows/package-lock.yml:23]: Why do we need `--legacy-peer-deps`?

File: craco.config.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// craco.config.js
```
Comment (Lukas-Batema) [inline craco.config.js:1]: What is CRACO? I never heard of it. I cannot approve this without knowing some stuff...

File: package.json [hunk 3]
```diff
@@ -36,7 +38,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "ghbuild": "CI=false react-scripts build",
+    "ghbuild": "CI=false craco build",
```
Comment (Lukas-Batema) [inline package.json:41]: Are you sure we are not losing any needed data, like when we compress an image?

**Review by Lukas-Batema — CHANGES_REQUESTED** [review]
Not sure how this works, or what it benefits, besides time, so I cannot approve

File: .github/workflows/package-lock.yml (hunk 1 above)
Comment (andrewstech) [reply .github/workflows/package-lock.yml:23]: > 

Ive got to fix a package

Comment (Lukas-Batema) [reply .github/workflows/package-lock.yml:23]: Ok, sounds fair.

File: craco.config.js (hunk 2 above)
Comment (andrewstech) [reply craco.config.js:1]: https://www.npmjs.com/package/@craco/craco

File: package.json (hunk 3 above)
Comment (andrewstech) [reply package.json:41]: Nop no data loss just better compiller

File: craco.config.js (hunk 2 above)
Comment (Lukas-Batema) [reply craco.config.js:1]: ✅

**Comment by andrewstech** [comment]
> Not sure how this works, or what it benefits, besides time, so I cannot approve

The load time of the site is increased plus the site reacts faster

**Comment by Lukas-Batema** [comment]
> > Not sure how this works, or what it benefits, besides time, so I cannot approve
> 
> The load time of the site is increased plus the site reacts faster

Anything besides time?

**Comment by andrewstech** [comment]
@win11bot approve

**Comment by Lukas-Batema** [comment]
Bruh

**Comment by Lukas-Batema** [comment]
I DO NOT APPROVE!!!

**Comment by Lukas-Batema** [comment]
> @win11bot approve

@win11bot Don't fall into the Darkside, resist the temptation to bypass non-approving reviews!

**Pull request merged**